tests/index.test.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-undef */
```

Comment:
Why did you need to disable this rule?

Reply:
Tha was an error on my part, I Have fixed that.